### PR TITLE
Remove everything related to editions

### PR DESCRIPTION
**Pull request by leo-geoffroy-sonarsource**

### CODE REVIEW

**Comment by pierre-guillot-sonarsource** [comment]
I pushed a fixup to remove `output-editions/` from `.gitignore`

**Review by pierre-guillot-sonarsource — APPROVED** [review]

File: sonar-update-center-mojo/src/main/java/org/sonar/updatecenter/mojo/GenerateMetadataMojo.java [hunk 1]
```diff
@@ -25,13 +25,6 @@
 import org.apache.maven.plugin.MojoFailureException;
```
Comment (aurelien-poscia-sonarsource) [inline sonar-update-center-mojo/src/main/java/org/sonar/updatecenter/mojo/GenerateMetadataMojo.java:25]: ultra-nitpick 😆 : cleanup import

File: .gitignore [hunk 2]
```diff
@@ -27,5 +27,4 @@ Thumbs.db
 # Folder config file
 Desktop.ini
 edition-templates.properties
```
Comment (aurelien-poscia-sonarsource) [inline .gitignore:29]: this line can be removed too

**Review by aurelien-poscia-sonarsource — APPROVED** [review]
2 little nitpicks, the rest LGTM :)

**Comment by ganncamp** [comment]
@pierre-guillot-sonarsource, @leo-geoffroy-sonarsource, @aurelien-poscia-sonarsource I've been trying to run the generation job for the last couple hours. I've finally got the metadata files cleaned up, and now I've hit
```
[ERROR] Failed to execute goal org.sonarsource.update-center:sonar-u ###### -properties/edition-templates.properties (No such file or directory) -> [Help 1]
org.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal org.sonarsource.update-center:sonar-u ###### -properties: Fail to execute mojo
...
Caused by: java.io.FileNotFoundException: /buddy/product-owner/sonar-u ###### -properties/edition-templates.properties (No such file or directory)
```

**Comment by pierre-guillot-sonarsource** [comment]
Hey @ganncamp, I think @leo-geoffroy-sonarsource fixed the deployment script. Can you confirm that the issue is resolved on your end? :)

**Comment by ganncamp** [comment]
Yes. I confirm. Thanks!

**Pull request merged**